addons/common/functions/fnc_exportMissionSQF.sqf

```diff
@@ -0,0 +1,381 @@
+        };
+
+        [_vehicle, _varName] call _fnc_processAttachedObjects;
+    };
+
+    _index
+};
+
+private _fnc_processStatic = {
+    params ["_object"];
+
+    private _index = _processedObjects find _object;
+
+    if (_index == -1) then {
+        _index = _processedObjects pushBack _object;
+
+        private _varName = VAR_OBJECT(_index);
+
+        _outputObjects pushBack ["%1 = createVehicle [%2, [0, 0, 0], [], 0, ""CAN_COLLIDE""];", _varName, str typeOf _object, getPosATL _object];
+        _outputObjects pushBack ["%1 setVectorDirAndUp %2;", _varName, [vectorDir _object, vectorUp _object]];
+        _outputObjects pushBack ["%1 setPosASL %2;", _varName, getPosASL _object];
+        _outputObjects pushBack ["%1 setDamage %2;", _varName, damage _object];
+
+        [_object, _varName] call _fnc_processInventory;
+        [_object, _varName] call _fnc_processAttachedObjects;
+
+        _outputObjects pushBack "";
+    };
+
+    _index
+};
+
+private _fnc_processObject = {
+    params ["_object"];
+
+    if (_object isKindOf "AllVehicles") then {
+        if (_object isKindOf "CAManBase") then {
+            _object call _fnc_processUnit
+        } else {
+            _object call _fnc_processVehicle
+        };
+    } else {
+        if (_object isKindOf "Thing" || {_object isKindOf "Static"}) then {
+            _object call _fnc_processStatic
+        };
+    };
+};
+
+private _objects = allMissionObjects "All";
+
+if (_radius > 0) then {
+    _objects = _objects inAreaArray [_position, _radius, _radius, 0, false, -1];
+};
+
+{
+    if (
+        alive _x
+        && {!isPlayer _x}
+        && {isNull attachedTo _x}
+        && {isNull isVehicleCargo _x}
+        && {isNull ropeAttachedTo _x}
+    ) then {
+        _x call _fnc_processObject
+    };
+} forEach _objects;
+
+if (_includeMarkers) then {
+    {
+        if (markerShape _x != "POLYLINE") then {
+            private _varName = format ["_marker%1", _forEachIndex];
+
+            _outputMarkers pushBack ["%1 = createMarker [%2, %3];", _varName, str _x, markerPos [_x, true]];
+            _outputMarkers pushBack ["%1 setMarkerDir %2;", _varName, markerDir _x];
+            _outputMarkers pushBack ["%1 setMarkerType %2;", _varName, str markerType _x];
+            _outputMarkers pushBack ["%1 setMarkerShape %2;", _varName, str markerShape _x];
+            _outputMarkers pushBack ["%1 setMarkerSize %2;", _varName, markerSize _x];
+            _outputMarkers pushBack ["%1 setMarkerText %2;", _varName, str markerText _x];
+            _outputMarkers pushBack ["%1 setMarkerBrush %2;", _varName, str markerBrush _x];
+            _outputMarkers pushBack ["%1 setMarkerColor %2;", _varName, str markerColor _x];
+            _outputMarkers pushBack ["%1 setMarkerAlpha %2;", _varName, markerAlpha _x];
+            _outputMarkers pushBack "";
+        };
+    } forEach allMapMarkers;
+};
+
+if (_addToCurators) then {
+    // todo
```

Comment:
ToDo

Reply:
Is this even needed? There is no option to change this in the module, and there is the "Auto Add Object" option that also does this.

Reply:
Idea was to have the outputted SQF add all created objects to curators independently of the `autoAddObjects` setting.

Reply:
Removed for now, can be added later if needed.